src/js/edu-benefits/5490/config/form.js

```diff
@@ -91,15 +89,58 @@ const formConfig = {
       pages: {
         applicantInformation: applicantInformation(fullSchema5490, {
           labels: { relationship: relationshipLabels }
-        })
+        }),
+        additionalBenefits: additionalBenefits(fullSchema5490, {
+          fields: ['civilianBenefitsAssistance', 'civilianBenefitsSource']
+        }),
+        applicantService: {
+          title: 'Applicant service',
```

Comment:
I didn't see anything suggesting we need to show titles where we weren't before, but @goldenmeanie could say for sure.